kustomize/overlays/example_cluster/nvidia-gpu-operator/README.md

```diff
@@ -0,0 +1,91 @@
+# GPU Operator
+
+## Install operator
+[Local operator install](../../../base/stable/nvidia-gpu-operator)
+
+## Building Custom vGPU Containers
+
+### vGPU Manager image
+
+Download vGPU software from the [NVIDIA Licensing Portal](https://nvid.nvidia.com/dashboard/#/dashboard)
+
+unzip the downloaded file
+```bash
+cd ~/Downloads
+unzip NVIDIA-GRID-Linux-KVM-535.54.06-535.54.03-536.25.zip
+```
+
+clone the driver container image repository
+```bash
+cd ~/git
+git clone https://gitlab.com/nvidia/container-images/driver
+cd driver/vgpu-manager
+cp -R rhel8 rhcos4.10
+cd rhcos4.10
+```
+
+Copy the downloaded driver into the os directory
+```bash
+cp ~/Downloads/Host_Drivers/*-vgpu-kvm.run ./
+```
+
+Build image
+```bash
+export PRIVATE_REGISTRY=my.private.registry:4567/nvidia-team/nvidia-images VERSION=535.54.06 OS_TAG=rhcos4.10 CUDA_VERSION=12.2.0
```

Comment:
Maybe write the my.private.registry to CHANGE_ME so it's easier to find.

Reply:
done